src/metax_api/tests/api/rest/base/views/datasets/write.py

```diff
@@ -440,6 +440,22 @@ class CatalogRecordApiWriteCreateTests(CatalogRecordApiWriteCommon):
     #
     #
 
+    def test_issued_date_is_generated(self):
+        ''' Issued date is generated for all but harvested catalogs if it doesn't exists '''
+        # ATT
```

Comment:
should be removed since we are testing also ida catalog?

Reply:
Good point